gradle.properties

```
@@ -26,7 +26,7 @@ dtestApiVersion=0.0.16
 # trunk is currently 5.1 - update when trunk moves
 dtestVersion=5.1
 dtestDependencyName=cassandra-dtest-local-all
-aswSdkVersion=2.20.43
+aswSdkVersion=2.26.12
```

There's a typo here:
```suggestion
awsSdkVersion=2.26.12
```